### ISSUE TITLE
Value of the Chanel Url field is not set the link tag in the RSS field and http head request support for Apple Podcasts

### ISSUE THREAD

**Issue by sdelice** [jgonyea/grav-plugin-podcast#30]
When I create a Podast channel Page, the value of the field Channel URL has no effect in the RSS field. By defautlt, it is set with the Grav Page Channel url but i want to add the exention .rss or .xml just like this rss feed : 

https://www.democracynow.org/podcast.xml

I really whant to have the control of the field because when i check the rss feed generate by grav-plugin-podcast witth castfeedvalidator.com it failed with the folowing errors : 
- Your RSS feed should be linked in the head of your podcast website, using a link tag.
- No website URL found for most recent episode. Did not finish website tests.
- We couldn't find any subscribe links on your feed webpage. We recommend at least including links to Apple Podcasts, Google Podcasts, and Subscribe On Android.
- This server does not support HTTP head requests. Head request support required for Apple Podcasts.

**Comment by jgonyea** [comment]
* Your RSS feed should be linked in the head of your podcast website, using a link tag.
I've updated the link tag to use the Channel Link field. Look for this in the next version.
* No website URL found for most recent episode. Did not finish website tests.
Added links for each episode. Look for this in the next version.
* We couldn't find any subscribe links on your feed webpage. We recommend at least including links to Apple Podcasts, Google Podcasts, and Subscribe On Android.
I'm going to punt on this until I can do a better job for the front end. Also seems to be an arbitrary "requirement"
* This server does not support HTTP head requests. Head request support required for Apple Podcasts.
I'm not sure what this is about, so I'll punt on this and research it more.
